boards/shields/ls0xx_generic/ls013b7dh03.overlay

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2020 Rohit Gujarathi
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&arduino_spi {
+	status = "okay";
+	cs-gpios = <&gpio0 20 GPIO_ACTIVE_HIGH>;
+
+	ls0xx@0 {
+		compatible = "sharp,ls0xx";
+		label = "LS0XX";
+		spi-max-frequency = <2000000>;
+		reg = <0>;
+		width = <128>;
+		height = <128>;
+		extcomin-gpios = <&gpio0 22 GPIO_ACTIVE_HIGH>; /* optional */
```

Comment:
Must be arduino_header, for example `extcomin-gpios = <&arduino_header 14 GPIO_ACTIVE_HIGH>; /* D8 */`.
Remove /* optional */, that is already described in binding file.